### PR TITLE
Build release debs in CI

### PR DESCRIPTION
**Pull request by wargio**
# SQUASH ME

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          [
+            "debian_buster",
+            "debian_stretch",
+            "debian_bullseye",
+            "debian_sid_php7.0",
+            "debian_sid_php7.1",
+            "debian_sid_php7.2",
+            "debian_sid_php7.3",
+            "debian_sid_php7.4",
+            "debian_sid_php8.0",
```
Comment (jvoisin) [inline .github/workflows/release.yml:29]: `debian_sid_php8.1` as well?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set timezone
+        if: startsWith(matrix.container, 'ubuntu:')
+        run: ln -snf /usr/share/zoneinfo/Europe/Paris /etc/localtime && echo Europe/Paris > /etc/timezone
+      - name: Update OS
+        run: apt-get -qqy update
+      - name: Install deps
+        run: DEBIAN_FRONTEND=noninteractive apt-get -qqy install fakeroot php-curl dpkg-dev gcc make libpcre3-dev dh-php ${{ matrix.dependencies }}
```
Comment (jvoisin) [inline .github/workflows/release.yml:90]: Might want to add `--no-install-recommends` as well I think

Comment (jvoisin) [reply .github/workflows/release.yml:90]: Also, why `php-curl`?

File: .github/workflows/release.yml (hunk 1 above)
Comment (wargio) [reply .github/workflows/release.yml]: yes, ofc.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      - name: Install deps
+        run: DEBIAN_FRONTEND=noninteractive apt-get -qqy --no-install-recommends install fakeroot php-xml php-curl dpkg-dev gcc make libpcre3-dev dh-php php-dev build-essential
+      - name: Build debs
+        run: DPKG_SKIP_TESTS=1 dpkg-buildpackage -i -us -uc -tc -I -rfakeroot
+      - name: Move built debs
+        run: mv ../*.deb ./
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: snuffleupagus-${{ matrix.name }}
```
Comment (jvoisin) [inline .github/workflows/release.yml:68]: Maybe `name: snuffleupagus-${{ matrix.name }}.deb`?

File: debian/rules [hunk 4]
```diff
@@ -27,5 +27,9 @@ override_dh_auto_install:
 	install -m 644 -o root -g root config/snuffleupagus.ini debian/snuffleupagus/etc/php/$$(/usr/bin/php-config --version | cut -d '.' -f1-2)/mods-available/snuffleupagus.ini
  
 override_dh_auto_test:
+ifeq ($(strip $(DPKG_SKIP_TESTS)),)
```
Comment (jvoisin) [inline debian/rules:30]: Create an issue to remove this :)

File: .github/workflows/release.yml (hunk 3 above)
Comment (wargio) [reply .github/workflows/release.yml:70]: no, is the name of the zip file.

File: debian/rules (hunk 4 above)
Comment (wargio) [reply debian/rules:30]: done.

**Pull request merged**